integration-tests/contract-manager-scenario.spec.ts

```diff
@@ -50,7 +50,9 @@ CONFIGS().forEach(({ lib, rpc, setup, knownBaker, knownContract }) => {
         await contract.methods.do(MANAGER_LAMBDA.transferImplicit("tz1eY5Aqa1kXDFoiebL28emyXFoneAoVg1zh", 50 * 1000000)).send({ amount: 0 })
         fail('Should throw during transfer with amount higher than balance')
       } catch (ex: any) {
-        expect(ex.message).toMatch('balance_too_low')
+        (protocol === Protocols.PtKathman) ? 
```

Comment:
I suppose that mondaynet and later protocols will also return subtraction_underflow?
Should we instead do a check for protocol === Protocols.PtJakart2 and use toContain rather than toMatch?

Reply:
I have changed it, so it checks for Jakarta for an exception. Otherwise, it will assume the new returned value.
I change to "ToContain" as well.